common/enablers/enableEpochsHandler.go

```diff
@@ -39,6 +41,330 @@ func (handler *enableEpochsHandler) EpochConfirmed(epoch uint32, _ uint64) {
 	handler.epochMut.Unlock()
 }
 
+// IsFlagDefined checks if a specific flag is supported by the current version of mx-chain-core-go
+func (handler *enableEpochsHandler) IsFlagDefined(flag core.EnableEpochFlag) bool {
+	switch flag {
+	case core.SCDeployFlag,
```

Comment:
can we instead initialize a map with all flags as keys (can be a global) and here just check that the flag is present in the map?

Reply:
will be done in the next PR when all flags will be moved here